### PR TITLE
Fix link to rasa-demo actions file in docs

### PR DESCRIPTION
**Pull request by melindaloubser1**
**Proposed changes**:
- fix link to rasa-demo actions file

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/user-guide/building-assistants.rst [hunk 1]
```diff
@@ -481,7 +481,7 @@ which will do something with the information the user has provided once the form
        return []
 
 In this case, we only tell the user that we’ll be in touch with them, however
-usually you would send this information to an API or a database. See the `rasa-demo <https://github.com/RasaHQ/rasa-demo/blob/master/demo/actions.py#L69>`_
+usually you would send this information to an API or a database. See the `rasa-demo <https://github.com/RasaHQ/rasa-demo/blob/master/actions/actions.py#L69>`_
```
Comment (erohmensing) [inline docs/user-guide/building-assistants.rst:484]:
```suggestion
usually you would send this information to an API or a database. See the `rasa-demo <https://github.com/RasaHQ/rasa-demo/blob/master/actions/actions.py#L71>`_
```

File: docs/user-guide/building-assistants.rst [hunk 2]
```diff
@@ -946,7 +946,7 @@ There are also a bunch of ways in which you can customise this policy. In Sara,
 we’ve customized it to suggest intents to the user within a certain confidence range to make
 it easier for the user to give the bot the information it needs.
 
-This is done by customizing the action ``ActionDefaultAskAffirmation`` as shown in the `Sara rasa-demo action server <https://github.com/RasaHQ/rasa-demo/blob/master/demo/actions.py#L443>`_
+This is done by customizing the action ``ActionDefaultAskAffirmation`` as shown in the `Sara rasa-demo action server <https://github.com/RasaHQ/rasa-demo/blob/master/actions/actions.py#L443>`_
```
Comment (erohmensing) [inline docs/user-guide/building-assistants.rst:949]:
```suggestion
This is done by customizing the action ``ActionDefaultAskAffirmation`` as shown in the `Sara rasa-demo action server <https://github.com/RasaHQ/rasa-demo/blob/master/actions/actions.py#L424>`_
```

**Comment by melindaloubser1** [comment]
Thanks for weighing in @vreyespue - sorry this got auto-merged before I saw your comments.

**Comment by melindaloubser1** [comment]
If you think it's easier to find the relevant code by linking to the method definition lines, feel free to open another PR and tag me in it.

**Comment by vreyespue** [comment]
Hi @melindaloubser1 
no problem, they were rather minor commentaries. Everything ok.
> If you think it's easier to find the relevant code by linking to the method definition lines, feel free to open another PR and tag me in it.

That's a good idea, I will take a look on that.
Many thanks!

**Comment by vreyespue** [comment]
Hi @melindaloubser1 
update: according to [this stackoverflow post](https://stackoverflow.com/questions/18220650/github-link-to-function-in-source), we have only two solutions: either we link to a certain revision of the file (`/{user}/{repo}/blob/{hash}/{file}#L{line}`), or we simply link to the whole file instead of the specific line.
Do you know any other solution? I did not find any way for linking (in general) to the method definition line. If you know how, where can I find an example and/or some explanation?
Many thanks!

**Comment by melindaloubser1** [comment]
@vreyespue No, unfortunately I don't know a better method - I meant with my suggestion just to change the line numbers. But thanks for finding out! Linking to a specific revision might be best, since the docs are just giving an example of an implementation, and don't promise that the code is the latest version or anything like that.

**Pull request merged**